WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -404,18 +410,29 @@ public void run() {
         ActivityId.trackLastActivity(ActivityId.POST_EDITOR);
     }
 
-    private void initializePostObjects(int localPostId) {
+    private void initializePostObjectsWithLocalId(int localPostId) {
         mPost = mPostStore.getPostByLocalPostId(localPostId);
         if (mPost != null) {
-            mOriginalPost = mPost.clone();
-            mPost = UploadService.updatePostWithCurrentlyCompletedUploads(mPost);
-            mMediaMarkedUploadingOnStartIds =
-                    AztecEditorFragment.getMediaMarkedUploadingInPostContent(this, mPost.getContent());
-            Collections.sort(mMediaMarkedUploadingOnStartIds);
-            mIsPage = mPost.isPage();
+            initializePostObject();
+        }
+    }
+
+    private void initializePostObjectsWithRemoteId(long remotePostId) {
```

Comment:
It seems we could get away with just adding a flag parameter (say, `isRemoteId`) to the original `initializePostObjects` to decide where which PostStore's method to use for loading. The separate functions seem to have only 1 line different between them.

Of course there's merit in separating into two functions to make their names more descriptive, but these methods are called by so few callers that I'm not sure it worths it. WDYT?

Reply:
I see your point - made changes in afe2ebc to have only the `initializePostObjects` not decide where to obtain the mPost from, but letting one-liners elsewhere to decide whether to use the remote or the local id depending on their context 👍